src/build.py

```diff
@@ -911,6 +912,8 @@ def Fastcomp():
        '-DLLVM_ENABLE_ASSERTIONS=ON'] + OverrideCMakeCompiler(),
       cwd=FASTCOMP_OUT_DIR, env=cc_env)
   proc.check_call(['ninja'], cwd=FASTCOMP_OUT_DIR, env=cc_env)
+  # Fastcomp has a different install location than the rest of the tools
+  BuildEnv(install_dir, bin_subdir=True)
```

Comment:
Yeah. Also if we kept the convention of "function names are always verbs" this would be more clear too, as this function both builds the runtime environment (i.e. the dlls) for the binaries, and returns environment vars to inject into the build environment.